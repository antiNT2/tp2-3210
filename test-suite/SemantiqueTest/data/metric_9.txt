num a;
bool b;


a = 1;

while (a + 1 * 3 < 10 + a) {
  b = -a < 1;
}