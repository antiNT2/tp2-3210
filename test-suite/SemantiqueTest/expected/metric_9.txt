{VAR:2, WHILE:1, IF:0, ENUM_VALUES:0, OP:6}